edx_rest_framework_extensions/jwt_decoder.py

```diff
@@ -0,0 +1,122 @@
+""" Utility functions. """
```

Comment:
**Note:** Not sure why git is treating this as new file.  This was renamed from utils.py to jwt_decoder.py.